src/components/DisplayCategories.jsx

```diff
@@ -2,33 +2,87 @@ import { getReviews } from "../requests";
 import { useState } from "react";
 import { Link } from "react-router-dom";
 import { useEffect } from "react";
+import { useSearchParams } from "react-router-dom";
+import moment from "moment";
 
-const DisplayCategories = ({ chosenCategory }) => {
+const DisplayCategories = ({
+  chosenCategory,
+  sortByOrder,
+  setSortByOrder,
+  sortByProperty,
+  setSortByProperty,
+}) => {
   const [categoryReviews, setCategoryReviews] = useState([]);
   const [loading, setLoading] = useState(true);
+  const [searchParams, setSearchParams] = useSearchParams();
+
   useEffect(() => {
-    getReviews().then((allReviews) => {
-      setCategoryReviews(
-        allReviews.filter((review) => {
-          return review.category === chosenCategory;
-        }),
-        setLoading(false)
+    getReviews(sortByOrder, sortByProperty, chosenCategory).then((data) => {
+      setSearchParams(
+        `order=${sortByOrder}&sort_by=${sortByProperty}&category=${chosenCategory}`
       );
+      setCategoryReviews(data);
+      setLoading(false);
     });
-  }, [chosenCategory]);
+  }, [chosenCategory, sortByOrder, sortByProperty]);
 
   if (loading === true) {
     return <section>Loading...</section>;
   }
   if (loading === false) {
     return (
-      <section className="specific_category_review">
+      <section>
+        <div>
+          order:
+          <button
+            onClick={() => {
+              setSortByOrder("ASC");
+            }}
+          >
+            Ascending
+          </button>
+          <button
+            onClick={() => {
+              setSortByOrder("DESC");
+            }}
+          >
+            Descending
+          </button>
+          order is: {sortByOrder}ENDING
+        </div>
+        <div>
+          order by:
+          <button
+            onClick={() => {
+              setSortByProperty("created_at");
+            }}
+          >
+            created_at
+          </button>
+          <button
+            onClick={() => {
+              setSortByProperty("votes");
+            }}
+          >
+            votes
+          </button>
+          <button
+            onClick={() => {
+              setSortByProperty("comment_count");
+            }}
+          >
+            comments
+          </button>
```

Comment:
DRY - some of these buttons are very similar, we could group them up and make the buttons programatically.  If we want to add another filter option later it would be very quick then!